### PR TITLE
Changed submit button on Act as user page to 'Act as user'

### PR DESCRIPTION
**Pull request by najclark**
## Motivation and Context

Changed title of submit button on Act as user page from 'sudo' to 'act as user' for clarity.

## How Has This Been Tested?

Before:
Screen Shot 2022-09-24 at 2 51 03 PM

After:
Screen Shot 2022-09-24 at 2 50 12 PM

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have run rubocop for style check. If you haven't, run `overcommit --install && overcommit --sign` to use pre-commit hook for linting

### CODE REVIEW

File: app/views/course_user_data/sudo.html.erb [hunk 1]
```diff
@@ -14,6 +14,6 @@
     <label for="student_autocomplete">Start typing student name or email</label>
   </div>
   <%= hidden_field_tag(:sudo_id) %>
-  <%= submit_tag "Sudo", {:value=>"Sudo", :class=> "btn primary"} %>
+  <%= submit_tag "Act as user", {:value=>"Act as user", :class=> "btn primary"} %>
```
Comment (damianhxy) [inline app/views/course_user_data/sudo.html.erb:17]: `:value=>...` seems to be redundant, so we can get rid of it.

Comment (damianhxy) [reply app/views/course_user_data/sudo.html.erb:17]: There also appears to be an unused sudo page under `/users/sudo.html.erb`, we can remove that as well.

**Review by damianhxy — APPROVED** [review]
Left some comments, but LGTM!

File: app/views/course_user_data/sudo.html.erb (hunk 1 above)
Comment (victorhuangwq) [reply app/views/course_user_data/sudo.html.erb:17]: @najclark

**Pull request merged**